shared/js/initApp.js

```diff
@@ -29,7 +29,7 @@ window.apps = {
       cdoSounds: CDOSounds,
       position: {blockYCoordinateInterval: 25},
       onInitialize: function() {
-        dashboard.createCallouts(this.callouts);
+        dashboard.createCallouts(this.level.callouts || this.callouts);
```

Comment:
Curious why later refs to level are `appOptions.level` while this is `this.level`?